tests/test_selection/test_check_estimator_selectors.py

```diff
@@ -24,6 +25,7 @@
         DropDuplicateFeatures(),
         DropCorrelatedFeatures(),
         SmartCorrelatedSelection(),
+        DropHighPSIFeatures(),
```

Comment:
there is 1 test that is failing, haven't gotten time to figure out why: check_fit2d_1sample